Content/Items/Gear/Weapons/Staff/LeadStaff.cs

```diff
@@ -0,0 +1,45 @@
+﻿using PathOfTerraria.Core.Items;
+using Terraria.ID;
+
+namespace PathOfTerraria.Content.Items.Gear.Weapons.Staff;
+
+internal class LeadStaff : Staff
+{
+	protected override int StaffType => ModContent.ProjectileType<LeadStaffHeld>();
+
+	public override void SetStaticDefaults()
+	{
+		base.SetStaticDefaults();
+
+		PoTStaticItemData staticData = this.GetStaticData();
+		staticData.DropChance = 1f;
+		staticData.MinDropItemLevel = 5;
+	}
+
+	public override void SetDefaults()
+	{
+		base.SetDefaults();
+
+		Item.shoot = ModContent.ProjectileType<LeadStaffProjectile>();
```

Comment:
Could this be updated to just use StaffHeldProjectile instead of making an extended class? Or were there intentions with it?

Reply:
I know they are abstracts but didn't see any changes, unless I missed something

Reply:
Fixed. Now it's a class that takes the staff ItemID, so there's only one held projectile. Thanks!